lib/diagnostic.js

```diff
@@ -7,25 +7,48 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+let words = paragraph.split(/\s+/);
+for (let i = 0; i < words.length; i++) {
+ words[i].replace(/\W+/, '');
+ normalizedWords[i] = words.toUpperCase();
+}
 };
 
```

Comment:
In order to use paragraph like we did in lab you need to define it here. JS has no idea what you mean by `paragraph` and therefore doesn't know how to call `.split` on it.